helper/resource.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * name : resource.js
+ * author : Ankit Shahu
+ * created-date : 10-April-2023
+ * Description : Resource level data for program and solution.
+ */
+
+
+const { ResourceType, SolutionType } = require("../common/enum.utils");
+const rp = require('request-promise');
+const { get } = require("request");
+const kendra_service = require("./kendra_service");
+
+exports.userExtensions = async function (req,res){
+    return new Promise(async function (resolve, reject) {
+        try {
+            let userExtensions = await kendra_service.getUserExtension(req.userDetails.token)
+            if(userExtensions.status === 200){
+                for(let platformCode = 0; platformCode < userExtensions.result.platformRoles.length; platformCode++){
```

Comment:
@ankitpws Should we not validate if the user has the given role for the program id given in request param?